src/control/system/member.go

```diff
@@ -139,15 +139,6 @@ func (ms MemberState) isTransitionIllegal(to MemberState) bool {
 		MemberStateJoined: {
 			MemberStateReady: true,
 		},
-		MemberStateStopping: {
-			MemberStateReady: true,
-		},
-		MemberStateExcluded: {
-			MemberStateReady:    true,
-			MemberStateJoined:   true,
-			MemberStateStopping: true,
-			MemberStateErrored:  true,
-		},
```

Comment:
Aren't some of these still illegal transitions? I think the only one that needs to be removed is MemberStateExcluded -> MemberStateStopping. Unless I've misunderstood the bug.

Reply:
These are unnecessary restrictions and while investigating the bug I've seen instances where these transitions are valid but prevented. I introduced these restrictions a while ago and the state machine has evolved since. There is no benefit in restricting transition from excluded or stopping states and allowing them offers greater usability.

Reply:
Okay, that makes sense. Thanks for explaining.